code/game/objects/items/toys.dm

```diff
@@ -917,6 +917,7 @@
 	icon = 'icons/obj/toys/balls.dmi'
 	icon_state = "baseball"
 	inhand_icon_state = "baseball"
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Uuuh, how's this related to this PR? Though, it's a one line change, so I don't mind as long as you add it to the changelog.